src/content/topics/integrations/partner-integrations/capabilities/feature-stores.mdx

```diff
@@ -31,20 +31,20 @@ In addition to the form variables defined in your manifest, you can use the spec
 
 ### The validation request property
 
-Specifying an optional validation request lets members to verify that they have properly filled out the details to correctly make a request.
+Specifying an optional validation request lets members verify that they have properly filled out the details to correctly make a request.
 
 The `parser` object lets LaunchDarkly interpret the response of the validation request. It allows a mapping of success and errors for the given response body of the request in the form of a [JSON pointer](https://datatracker.ietf.org/doc/html/rfc6901). The `parser` object has two properties: a required `success` and an optional `error`.
 
-Here is an example `parse` object:
+Here is an example `parser` object:
 
```

Comment:
Since the previous paragraph says `parser`, I figure that's what this is supposed to say. I don't know for sure. 

Tried to also fix the code formatting.